contract_operating_unit/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 ForgeFlow S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Contract Operating Unit",
+    "version": "12.0.1.0.0",
+    "license": "LGPL-3",
+    "author": "ForgeFlow S.L., " "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/operating-unit",
+    "category": "Invoicing",
+    "depends": ["contract", "operating_unit"],
```

Comment:
I think this should depends on accounting, otherwise, the invoice created will has no OU ?
```suggestion
    "depends": ["contract", "account_operating_unit"],
```